dbf2csv/models/dbf2csv.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 INGETIVE (<https://ingetive.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import csv
+import base64
+import logging
+from odoo import models, fields, api
+
+try:
+    from dbfpy import dbf
+except ImportError:
+    dbfpy = None
```

Comment:
I suggest to log the import error as per [OCA guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror).

Reply:
Ok, thanks